ssv/spectest/tests/runner/duties/newduty/decided.go

```diff
@@ -36,34 +36,38 @@ func PostDecided() tests.SpecTest {
 		Tests: []*StartNewRunnerDutySpecTest{
 			{
 				Name:                    "sync committee aggregator",
-				Runner:                  finishRunner(testingutils.SyncCommitteeContributionRunner(ks), &testingutils.TestingSyncCommitteeContributionNexEpochDuty),
+				Runner:                  decidedRunner(testingutils.SyncCommitteeContributionRunner(ks), &testingutils.TestingSyncCommitteeContributionNexEpochDuty),
 				Duty:                    &testingutils.TestingSyncCommitteeContributionNexEpochDuty,
-				PostDutyRunnerStateRoot: "0ae01aec6a06e42b300b5de874574a19ac2c27a0a32f0ec77f3769a78392065d",
+				PostDutyRunnerStateRoot: postDecidedSyncCommitteeContributionSC().Root(),
+				PostDutyRunnerState:     postDecidedSyncCommitteeContributionSC().ExpectedState,
 				OutputMessages: []*types.SignedPartialSignatureMessage{
 					testingutils.PreConsensusContributionProofNextEpochMsg(ks.Shares[1], ks.Shares[1], 1, 1), // broadcasts when starting a new duty
 				},
 			},
 			{
 				Name:                    "sync committee",
-				Runner:                  finishRunner(testingutils.SyncCommitteeRunner(ks), &testingutils.TestingSyncCommitteeDuty),
+				Runner:                  decidedRunner(testingutils.SyncCommitteeRunner(ks), &testingutils.TestingSyncCommitteeDuty),
 				Duty:                    &testingutils.TestingSyncCommitteeDuty,
```

Comment:
We are not using `testingutils.TestingSyncCommitteeDuty*NextEpoch*` here like the rest of the tests.
From my understanding of the tests, a good happy flow should look like the following:
```
	Runner:                  decidedRunner(testingutils.SyncCommitteeRunner(ks), &testingutils.TestingSyncCommitteeDuty),
	Duty:                    &testingutils.TestingSyncCommitteeDuty*NextEpoch*
```
This should startup with a decided runner for some epoch, and hten start a duty for the next.
But none of the tests follow this pattern.

On the other hand maybe we want to show that it doesn't matter for the spec if we follow the happy flow.
Then we should just have all tests be consistent..

wdyt @alonmuroch?

Reply:
moving discussion to #271